metrics/errors.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  name: 'enables JavaScript Sentry error metrics',
+  startDate: '2015-01-01',
+  subjectAttributes: ['uuid', 'sampleRate'],
+  independentVariables: ['sentryEnabled'],
+  eligibilityFunction: function (subject) {
+    return !! subject.sampleRate && subject.uuid && this.bernoulliTrial(subject.sampleRate, subject.uuid)
```

Comment:
Thanks for the explanation @dannycoates. I'm not sold on setting / hardcoding the `1.0 : 0.3` values in this repository and I'd rather have them be in the content server configuration. However I will trust your advice on this. @shane-tomlinson thoughts on setting these values here?